db/__init__.py

```diff
@@ -0,0 +1,142 @@
+"""
+
+    Reynir: Natural language processing for Icelandic
+
+    Scraper database model
+
+    Copyright (C) 2019 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+
+
+    This module contains database-related functionality.
+
+"""
+
+import platform
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from settings import Settings, ConfigError
+
+from sqlalchemy.exc import SQLAlchemyError as SqlError
+from sqlalchemy.exc import IntegrityError as SqlIntegrityError
+from sqlalchemy.exc import DataError as SqlDataError
+from sqlalchemy import desc as SqlDesc
+from sqlalchemy import func as dbfunc
+
+from .models import Base
+
+# Allow client use of these exception without importing them from sqlalchemy
+IntegrityError = SqlIntegrityError
+DatabaseError = SqlError
+DataError = SqlDataError
+# Same for the desc() function
+desc = SqlDesc
```

Comment:
Is the previous code still needed now that this is in `__init__.py`? Isn't it simply enough to do `from sqlalchemy.exc import XXX as YYY`?